### ISSUE TITLE
unknown benefit_type on request issues

### ISSUE THREAD

**Issue by pkarman** [department-of-veterans-affairs/caseflow#10945]
There are currently 9 request issues in production with a benefit_type of "unknown". They were all created since 16 May.

**Comment by pkarman** [comment]
All but 1 are `duplicate_of_rating_issue_in_active_review`

**Comment by pkarman** [comment]
The remaining 1 had a parent contested decision issue, and the benefit type should inherit from there. h/t @leikkisa

**Comment by leikkisa** [comment]
The ineligible ones had contested decision issues also, so I updated these to those benefit types, matching Peter's PR.  Here are their request issue IDs:

[57957, 62176, 52540, 58898, 55049, 59690, 63854, 70329]

Moving this ticket to in validation.

While investigating this, I noticed that all of these were ineligible due to the same "issue in active review", even though they are all coming from different contested decision issues.  Opening another ticket for that.
